crates/testing/src/predicates.rs

```diff
@@ -76,3 +89,10 @@ pub fn decided_upgrade_cert() -> Predicate<ConsensusTaskTestState> {
         "expected decided_upgrade_cert to be Some(_)",
     )
 }
+
+pub fn is_at_view_number(n: u64) -> Predicate<ConsensusTaskTestState> {
```

Comment:
This allows us to easily verify that the view is at a particular node leader for a given step, which is helpful for debugging but also knowing when a proposal happens and enforcing who the leader was during the proposal.